src/client/pages/ConsentsData.tsx

```diff
@@ -37,51 +88,55 @@ const marketingText = css`
   margin-top: ${space[4]}px;
 `;
 
-export const ConsentsData = ({ consented, description }: ConsentsDataProps) => {
+export const ConsentsData = ({ id, consented, name }: ConsentsDataProps) => {
   const autoRow = getAutoRow(1, gridItemColumnConsents);
-
-  const label = <span css={checkboxLabel}>{description}</span>;
+  const autoSwitchRow = getAutoRow(1, gridItemToggleSwitch);
 
   return (
     <ConsentsLayout title="Your data" current={CONSENTS_PAGES.YOUR_DATA}>
-      {description && (
+      {name && (
         <ConsentsContent>
-          <h2 css={[heading, greyBorderTop, autoRow()]}>
-            We never share your data without your permission
+          <h2 css={[heading, removeMargin, greyBorderTop, autoRow()]}>
+            What we mean by your data
           </h2>
-          <p css={[text, autoRow()]}>
-            We think carefully about our use of personal data and use it
-            responsibly. We have a team who are dedicated to keeping any data we
-            collect safe and secure. You can find out more about how The
-            Guardian aims to safeguard users data by going to the Privacy
-            section of the website.
-          </p>
-          <h2 css={[heading, headingMarginSpace6, autoRow()]}>
-            Using your data for marketing analysis
-          </h2>
-          <p css={[text, autoRow()]}>
-            From time to time we may use your personal data for marketing
-            analysis. That includes looking at what products or services you
-            have bought from us and what pages you have been viewing on
-            theguardian.com and other Guardian websites (e.g. Guardian Jobs or
-            Guardian Holidays). We do this to understand your interests and
-            preferences so that we can make our marketing communication more
-            relevant to you.
-          </p>
-          <div css={autoRow()}>
+          <ul css={[text, listBullets, autoSwitchRow()]}>
+            <li>Information you provide e.g. email address</li>
+            <li>Products or services you buy from us</li>
+            <li>
+              Pages you view on theguardian.com or other Guardian websites when
+              signed in
+            </li>
+          </ul>
+
+          <fieldset css={[switchRow, greyBorderTop, autoSwitchRow()]}>
+            <ToggleSwitchInput
+              id={id}
+              // TODO replace with Consent.name once IDAPI model is updated
```

Comment:
Release plan: 
1/ release Gateway and Manage-Frontend changes first with hard coded new consent wording
2/ Update Identity-Model
3/ Update Gateway and Manage-Frontend to use label wording from model